app/models/pay/webhook.rb

```diff
@@ -8,7 +8,7 @@ def process!
       Pay::Webhooks.instrument type: "#{processor}.#{event_type}", event: rehydrated_event
 
       # Remove after successfully processing
-      destroy
+      # destroy
```

Comment:
I was wondering why the addition of lemon_squeezy is commenting this line of code.